lib/resources/example_json/r4_examples_financial_transactions.rb

```diff
@@ -0,0 +1,3744 @@
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method",
+                "valueCodeableConcept": {
+                    "coding": [
+                        {
+                            "code": "CHCK",
+                            "display": "Check",
+                            "system": "http://terminology.hl7.org/CodeSystem/v2-0570",
+                            "userSelected": true
+                        }
+                    ],
+                    "text": "Check"
+                }
+            },
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount",
+                "valueMoney": {
+                    "value": "123000",
+                    "currency": "USD"
+                }
+            },
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-payment-date",
+                "valueDate": "2020-02-01"
+            },
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-payment-account-number",
+                "valueString": "1234"
+            },
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation",
+                "extension": [
+                    {
+                        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-target",
+                        "valueReference": {
+                            "reference": "Encounter/31363178"
+                        }
+                    },
+                    {
+                        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-amount",
+                        "valueMoney": {
+                            "value": "400000",
+                            "currency": "USD"
+                        }
+                    }
+                ]
+            },
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias",
+                "extension": [
+                    {
+                        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-transaction-alias-alias",
+                        "valueString": "1300"
+                    }
+                ]
+            },
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-location",
+                "valueString": "201802160003"
+            },
+            {
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount",
+                "valueMoney": {
+                    "value": "100000",
+                    "currency": "USD"
+                }
+            }
+        ]
+    }
+    R4_FINANCIAL_TRANSACTION ||={
```

Comment:
Yes removed this. Since this file got renamed the new examples.rb does not have the ones which are not required. https://github.com/cerner/fhir.cerner.com/pull/297/commits/a1a39411fbb35b2ccf79e521ffae5e6ae92d62b9#diff-f23ddfd1e13f40ced360def6aac5c0b5L1

Reply:
+1